src/main/java/uk/gov/hmcts/reform/hmc/client/futurehearing/HearingManagementInterfaceResponse.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.hmc.client.futurehearing;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HearingManagementInterfaceResponse {
+
+    @JsonProperty("response code")
```

Comment:
Do we need this line (same for `description`)? If so, I guess there's a missing underscore in `response_code`